tests/app/main/views/test_index_page.py

```diff
@@ -42,3 +42,18 @@ def test_add_buyer_email_domain_link_is_shown_to_users_with_right_roles(self, ro
 
         assert link_is_visible is link_should_be_visible, \
             "Role {} {} see the link".format(role, "can not" if link_should_be_visible else "can")
+
+    @pytest.mark.parametrize("role, link_should_be_visible", [
+        ("admin", False),
+        ("admin-ccs-category", False),
+        ("admin-ccs-sourcing", False),
+        ("admin-manager", True),
+    ])
+    def test_manage_admin_users_link_is_shown_to_users_with_the_right_role(self, role, link_should_be_visible):
+        self.user_role = role
+        response = self.client.get('/admin')
+        data = response.get_data(as_text=True)
+        link_is_visible = "Manage users" in data
+
```

Comment:
https://www.python.org/dev/peps/pep-0008/#maximum-line-length
Generally parenthesis are preferred over line continuation characters.
```
assert (
    link_is_visible is link_should_be_visible,
    "Role {} {} see the link".format(role, "can not" if link_should_be_visible else "can"
)
```

Reply:
Damn, I tried this but Flake8 is confused by it...
`F999 assertion is always true, perhaps remove parentheses?`

Reply:
Ah found it:
```
assert link_is_visible is link_should_be_visible, (
    "Role {} {} see the link".format(role, "can not" if link_should_be_visible else "can"
)
```

Reply:
Weird assertion syntax